packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -58,14 +58,16 @@ const StyledLink = styled.a`
   }
 
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    ${({ script }) => script && getGreatPrimer(script)};
+    ${({ script, typography }) =>
+      script && typography !== 'pica' && getGreatPrimer(script)}
   }
 `;
 
-const ALIGN_RANK_TO_LINK_SPACING = '0.375rem';
+const ALIGN_RANK_TO_LINK_SPACING = rankTypography =>
```

Comment:
This casing is typically used to denote a constant value. As it's now a function, it might be more appropriate to revert to camelCase and also rename it to something like `getItemSpacing`.